lib/cli/src/Cardano/CLI.hs

```diff
@@ -367,114 +377,139 @@ runCli = join . customExecParser preferences
 
 cmdKey :: Mod CommandFields (IO ())
 cmdKey = command "key" $ info (helper <*> cmds) $ mempty
-    <> progDesc "Derive keys from mnemonics."
+    <> progDesc "Derive and manipulate keys."
+    <> headerDoc (Just $ string $ mconcat
+        [ "Keys can be passed as arguments or read as standard input. Both "
+        , "bech32- and hexadecimal encodings are supported."
+        , "\n\n"
+        , "For instance:\n"
+        , "$ cardano-wallet-byron key root --wallet-style icarus --encoding bech32 -- express theme celery coral permit ... \\\n"
```

Comment:
We should avoid hard-coding the name of the binary here, since it actually depends on where this command is mounted. I'd suggest to simply stick with the commands:

```
key root --wallet-style icarus ...
```

Reply:
👍